xoinvader/render.py

```diff
@@ -15,12 +15,15 @@ class Renderable(object):
 
     .. class-variables::
 
+    * compound (bool): if object consists of other renderables
+
     * render_priority (int): priority for renderer, greater -> rendered later
 
     * draw_on_border (bool): allow or not drawing on border
     if not allowed - renderer will send remove_obsolete signal to object
     """
 
+    compound = False
```

Comment:
Such change leads to 2 extra function calls and logging empty list for Background and GUI elements in one moment and for charges every shot. This is not so bad but I don't see any needs to do so (now).